ElisBot/ElisBot.py

```diff
@@ -186,12 +186,12 @@ def handle(msg):
             for user in response:
                 nome = user['Nome']
                 cognome = user['Cognome']
-                cn = user["Capo_Nucleo"]
-                cs = user["Capo_Stanza"]
+                capoNucleo = user["Capo_Nucleo"]
```

Comment:
questa sintassi non va bene per Python, bisogna usare lo snake_case, quindi capo_nucleo in caso di parole "composte"